client/lib/purchases/index.js

```diff
@@ -57,14 +57,18 @@ function hasPrivateRegistration( purchase ) {
 }
 
 function isCancelable( purchase ) {
-	if ( isRefundable( purchase ) ) {
```

Comment:
It's moved below. We can't refund expired and included purchases. In long run we should patch backend.